lib/shared/addon/utils/util.js

```diff
@@ -427,6 +427,10 @@ export function keysToCamel(obj) {
   return obj;
 }
 
+export function deepCopy(obj) {
+  return JSON.parse(JSON.stringify(obj));
```

Comment:
Now only one place is using it https://github.com/rancher/ui/blob/fa855578163a4bd3788860cd0a0aff5fdc4c6238/lib/pipeline/addon/components/pipeline-steps/component.js#L34

It can be serialized. Thanks!